assets/sass/components/settings/_googlesitekit-settings-module.scss

```diff
@@ -184,6 +189,11 @@
 		&--wrap {
 			word-break: break-all;
 		}
+
+		&--tiny a {
+			font-size: 0.8125rem;
+			font-weight: 400;
+		}
```

Comment:
This works, but I think conceptually it would make more sense to apply this styling to the entire copy (i.e. even if it wasn't only a link). Otherwise it would look weird having a text there with a link in it, as the link would randomly appear smaller than the rest of the text around it.